Makefile

```diff
@@ -150,6 +152,17 @@ ifneq ($(HAS_SERVER),)
 	$(GO) tool cover -html=server/coverage.txt
 endif
 
+## Extract strings for translation from the source code.
+.PHONY: i18n-extract
+i18n-extract:
+ifneq ($(HAS_WEBAPP),)
+ifeq ($(HAS_MM_UTILITIES),)
+	@echo "You must clone github.com/mattermost/mattermost-utilities repo in .. to use this command"
```

Comment:
Out of scope for this PR, but I wonder if we could go get this instead, or perhaps build it as a dependency into the `manifest` tool somehow. Also, I guess we don't support translating strings embedded in the Go source?

Reply:
> Out of scope for this PR, but I wonder if we could go get this instead, or perhaps build it as a dependency into the `manifest` tool somehow. 

Maybe we can do this. But to me translating `webapp` is a rare case. And installing `mattermost-utilities` not to big of a task. Feel free to open an issue for this.

> Also, I guess we don't support translating strings embedded in the Go source?

`i18n-extract` only extracts the strings in `webapp` ATM. There is an ongoing discussion about how server translations should work. See https://github.com/mattermost/mattermost-server/pull/10949.